fe2-android/app/src/main/java/com/github/dedis/student20_pop/detail/fragments/ManageElectionFragment.java

```diff
@@ -75,43 +113,229 @@ public View onCreateView(
+        });
+
+        editQuestionBuilder.create();
+
+        // On click, edit new name button
+
+        editName.setOnClickListener(v -> {
+            inputName.setText(null); // we make sure the text is blank when we reclick the button
+            if (inputName.getParent() != null) {
+                ((ViewGroup) inputName.getParent()).removeView(inputName);
+            }
+            editNameBuilder.setView(inputName);
+
+            editNameBuilder.show();
+        });
+
+        // On click, edit new question button
+
+        editQuestion.setOnClickListener(v -> {
+            inputQuestion.setText(null); // we make sure the text is blank when we click the button
+            if (inputQuestion.getParent() != null) {
+                ((ViewGroup) inputQuestion.getParent()).removeView(inputQuestion);
+            }
+            editQuestionBuilder.setView(inputQuestion);
+            editQuestionBuilder.show();
+        });
+
+        // On click, edit start time button
+        editStartTimeButton.setOnClickListener(
                 v -> {
-                   // TODO implement the cancel election action with backend when the backend is ready
-                    laoDetailViewModel.terminateCurrentElection();
-                    laoDetailViewModel.openLaoDetail();
+                    // we set the request code
+                    requestCode = START_TIME_CODE;
+                    // show the timePicker
+                    timePickerDialog.show();
+                });
 
+        // On click, edit end time button
+        editEndTimeButton.setOnClickListener(
+                v -> {
+                    // we set the request code
+                    requestCode = END_TIME_CODE;
+                    // show the timePicker
+                    timePickerDialog.show();
                 });
 
-        // Subscribe to "open home" event
-        laoDetailViewModel
-                .getOpenHomeEvent()
-                .observe(
-                        this,
-                        booleanEvent -> {
-                            Boolean event = booleanEvent.getContentIfNotHandled();
-                            if (event != null) {
-                                setupHomeActivity();
-                            }
-                        });
+        // On click, edit start date button
+        editStartDateButton.setOnClickListener(
+                v -> {
+                    // we set the request code
+                    requestCode = START_DATE_CODE;
+                    // show the timePicker
+                    datePickerDialog.show();
+                });
 
+        // On click, edit end time button
+        editEndDateButton.setOnClickListener(
+                v -> {
+                    // we set the request code
+                    requestCode = END_DATE_CODE;
+                    // show the timePicker
+                    datePickerDialog.show();
+                });
 
-    }
-    public void setupHomeButton() {
```

Comment:
Why is the access to home from this fragment deleted?

Reply:
This setupHomeButton was a mistake as this button is already set in the LaoDetailActivity